understanding/20/use-of-color.html

```diff
@@ -38,13 +38,28 @@ <h2>Intent of Use of Color</h2>
          only in color.
       </p>
       
-      <p>Examples of information conveyed by color differences: “required fields are red",
-         “error is shown in red", and “Mary's sales are in red, Tom's are in blue". Examples
-         of indications of an action include: using color to indicate that a link will open
-         in a new window or that a database entry has been updated successfully. An example
-         of prompting a response would be: using highlighting on form fields to indicate that
-         a required field had been left blank.
-      </p>
+      <p>Examples of common failures include:</p>
+      <ul>
+        <li>
+            <strong>Information conveyed by color differences:</strong> 
+            <ul>
+                <li>required fields are only identified by changing their existing border to a red color;</li>
+                <li>the type of transaction is only visually distinguished by using different foreground colors: red for debits and green for credits;</li>
+                <li>subway lines are visually distinguished solely by employing various filled colors;</li>
+            </ul>
+        </li>
+        <li>using color alone to indicate
+         that a link will open in a new window or that a database entry has been updated successfully.</li>
```

Comment:
> using color alone to indicate that a link will open in a new window

Has anyone ever seen this implemented? The example seems quite artificial.

Reply:
happy to take it out. i did come across something like this on a site ages ago, but yeah not very common